x-pack/legacy/plugins/siem/public/components/ml_popover/api.tsx

```diff
@@ -96,13 +131,17 @@ export const setupMlJob = async ({
  *
  * @param datafeedIds
  * @param start
- * @param headers
+ * @param headers optional headers to add
  */
-export const startDatafeeds = async (
-  datafeedIds: string[],
-  headers: Record<string, string | undefined>,
-  start = 0
-): Promise<StartDatafeedResponse> => {
+export const startDatafeeds = async ({
+  datafeedIds,
+  headers,
+  start = 0,
+}: {
+  datafeedIds: string[];
+  start: number;
+  headers?: Record<string, string | undefined>;
```

Comment:
I think if you look closely at the type for fetch below you can change this to remove the `undefined` as a value:

```suggestion
  headers?: Record<string, string>;
```

Here is what the type looks like:
<img width="758" alt="Screen Shot 2019-10-14 at 5 27 01 PM" src="https://user-images.githubusercontent.com/1151048/66789247-08bc8600-eea8-11e9-97a5-0f6c38688598.png">

Removing it didn't cause issues on my side. Should they all be changed for both request and response to be just `Record<string, string>;`?

Reply:
Ah yeah, nice catch! Those are definitely lingerers. There's another in there I'll remove as well. Thanks! 🙂